manifests/cinder-csi-plugin/cinder-csi-nodeplugin.yaml

```diff
@@ -63,6 +63,7 @@ spec:
             - "--nodeid=$(NODE_ID)"
             - "--endpoint=$(CSI_ENDPOINT)"
             - "--cloud-config=$(CLOUD_CONFIG)"
+            - "--v=5"
```

Comment:
previously , I was told in different PR this should not be set to verbose by default as user always has capability to do so by default we should not set log level..

Reply:
@jichenjc Thanks for pointing out. I just added to debug the failures before. Missed to remove it. Ill update